README.md

```diff
@@ -1,18 +1,16 @@
 # syrupy
 
-<span><img align="right" width="200" height="200" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo"></span>
+<img align="center" width="100px" height="100" src="https://user-images.githubusercontent.com/2528959/69500147-85d71400-0ec6-11ea-867a-277881278e57.png" alt="Logo">
```

Comment:
This does not seem to align center when rendered.

Reply:
removed redundant alignment